src/types/BuiltInInterfaceAdder.ts

```diff
@@ -0,0 +1,86 @@
+import type { BRSInterfaceData } from '../roku-types';
+import { components, interfaces } from '../roku-types';
+import { Cache } from '../Cache';
+import type { TypedFunctionType } from './TypedFunctionType';
+import { SymbolTypeFlag } from '../SymbolTable';
+import type { BscType } from './BscType';
+import { isArrayType, isBooleanType, isCallableType, isClassType, isDoubleType, isEnumMemberType, isEnumType, isFloatType, isIntegerType, isInvalidType, isLongIntegerType, isStringType } from '../astUtils/reflection';
+
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
```

Comment:
Is this still necessary?

Reply:
It's a pure static class... I think that's why.